ci/nits.sh

```diff
@@ -18,20 +18,23 @@ declare prints=(
 
 # Parts of the tree that are expected to be print free
 declare print_free_tree=(
-  'core/src'
-  'faucet/src'
-  'ledger/src'
-  'metrics/src'
-  'net-utils/src'
-  'runtime/src'
-  'sdk/bpf/rust/rust-utils'
-  'sdk/src'
-  'programs/bpf/rust'
-  'programs/stake/src'
-  'programs/vote/src'
+  ':core/src/**.rs'
+  ':faucet/src/**.rs'
+  ':ledger/src/**.rs'
+  ':metrics/src/**.rs'
+  ':net-utils/src/**.rs'
+  ':runtime/src/**.rs'
+  ':sdk/bpf/rust/rust-utils/**.rs'
+  ':sdk/**.rs'
+  ':programs/bpf/rust/**.rs'
+  ':programs/stake/src/**.rs'
+  ':programs/vote/src/**.rs'
+  ':^**bin**.rs'
+  ':^**bench**.rs'
+  ':^**test**.rs'
 )
 
-if _ git --no-pager grep -n --max-depth=0 "${prints[@]/#/-e }" -- "${print_free_tree[@]}"; then
```

Comment:
It there some reason `--max-depth=0` is added by [here?](https://github.com/solana-labs/solana/pull/4137/files#diff-47601a2b7c12e7cddcedbd2c4bb20104R28)

Reply:
Odd enough, I needed to remove the space (` `) after `-e` in my environment by running `ci/nits.sh` to make the detection actually work, although it worked as intended by directly replaying the quoted `grep` invocation. And I have no interest in digging into the rabbit hole of `bash` escaping voodoo this time. :)